src/locales/en/messages.po

```diff
@@ -667,8 +671,8 @@ msgid "Configure your project's tokenomics."
 msgstr "Configure your project's tokenomics."
 
 #: src/components/v2/V2Project/NewDeployModal.tsx
-msgid "Congratulations on launching your project! We recommend you take these next steps to help your project gain traction. These steps are optional and you can complete them later on."
-msgstr "Congratulations on launching your project! We recommend you take these next steps to help your project gain traction. These steps are optional and you can complete them later on."
+msgid "Congratulations on launching your project! Continue with these optional next steps."
```

Comment:
This seems like a bit of a contradiction to me - 'Continue with these optional...'. WDYT about 'The next steps are optional and can be completed at any time.'?

Reply:
yep sgtm